src/methods/broadcast_tx_async.rs

```diff
@@ -1,3 +1,106 @@
+//! Sends asynchronous transactions.
+//!
+//! Sends a signed transaction to the RPC and returns the transaction hash.
+//!
+//! ## Example
+//!
+//! ```
```

Comment:
Wow, this is quite long. Plus, we can't realistically test submitting transactions on-chain here. Let's strip this to showing just how to construct a request, without sending it to the client. By doing that, we can clean up a lot of boilerplate.

Might also help to include a link to the example that shows how to use this method - `contract_change_method`.